### DISCUSSION TITLE
Remove the Centurion (AC-500)

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#133] [Ideas]
I don't think it makes sense to have 2 AA-focused experimental units in the same faction, even if one is a lot heavier than the other.
This is also one of those rare cases where the other unit both looks and feels better to use than the BrewLan unit.

**Comment by PurpleMossClump** [comment]
Afaik, the unit doesn't even function properly. I haven't built it often, but when I have, it didn't seem like it was able to use all of its weapons.

**Comment by Kami-Interface** [comment]
Yeah, they're mounted in different places, and it will hover/circle its target at a distance where not all weapons will fire I think. Baltazar could probably fix such an issue for us, but even so I don't think it's worth keeping this unit around
